app/src/main/java/com/khinthirisoe/bakingapp/di/component/AppComponent.kt

```diff
@@ -17,4 +18,6 @@ interface AppComponent {
     fun inject(application: App)
 
     fun app(): App
+
+    fun apiService(): ApiHelper
```

Comment:
@KhinThiriSoe keep the name consistent. 
you can either define **apiService(): ApiService** or **apiHelper(): ApiHelper**